src/raftstore/store/util.rs

```diff
@@ -519,8 +538,38 @@ impl Lease {
     }
 
     pub fn expire(&mut self) {
+        self.expire_remote_lease();
         self.bound = None;
     }
+
+    pub fn expire_remote_lease(&mut self) {
+        // Expire remote lease if there is any.
+        if let Some(r) = self.remote.take() {
+            r.expire();
+        }
+    }
+
+    pub fn maybe_new_remote_lease(&mut self, term: u64) -> Option<RemoteLease> {
+        if self.remote.is_some() {
+            // At most one connected RemoteLease.
+            return None;
+        }
+        let expired_time = match self.bound {
+            Some(Either::Right(ts)) => timespec_to_u64(ts),
```

Comment:
Should `last_update` be updated here?

Reply:
We dont need to update here. It should be updated when renew lease.